architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,194 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization policy determines whether a user is allowed to perform a given
```

Comment:
Hmm starting to think this is done on purpose. Is there a reason why there's no determiner before "authorization policy"? So a _the_/_a_/_an_? In this case I'd suggest an _an_. 

http://grammar.yourdictionary.com/parts-of-speech/nouns/what/what-is-a-determiner.html